frontend/fingertips-frontend/components/molecules/AreaSuggestionPanel/index.tsx

```diff
@@ -90,7 +91,23 @@ export const AreaAutoCompleteSuggestionPanel = ({
         SearchParams.AreaTypeSelected,
         selectedAreaTypeKey
       );
+
+      if (selectedAreaCode === areaCodeForEngland) {
+        stateManager.addParamValueToState(
+          SearchParams.GroupTypeSelected,
+          englandAreaType.key
+        );
+        stateManager.addParamValueToState(
+          SearchParams.GroupSelected,
+          areaCodeForEngland
+        );
+        stateManager.addParamValueToState(
+          SearchParams.GroupAreaSelected,
+          'ALL'
```

Comment:
We have a const for this 
`import { ALL_AREAS_SELECTED } from './areaFilterHelpers/constants';`